### PR TITLE
drop the DIST (ubi9) suffix in image tags

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

**Review by shivamerla — APPROVED** [review]
lgtm

File: Makefile [hunk 1]
```diff
@@ -278,13 +276,9 @@ else
 # We skip this for the development release
 DEVEL_RELEASE_IMAGE_VERSION ?= devel
 ifneq ($(strip $(VERSION)),$(DEVEL_RELEASE_IMAGE_VERSION))
-push-$(DEFAULT_PUSH_TARGET): push-short
+push-$(DEFAULT_PUSH_TARGET):
```
Comment (cdesiniotis) [inline Makefile]: What is the purpose of this now? Can we remove this?

File: validator/Makefile [hunk 2]
```diff
@@ -68,13 +66,9 @@ validator:
 # We skip this for the development release
 DEVEL_RELEASE_IMAGE_VERSION ?= devel
 ifneq ($(strip $(VERSION)),$(DEVEL_RELEASE_IMAGE_VERSION))
-push-$(DEFAULT_PUSH_TARGET): push-short
+push-$(DEFAULT_PUSH_TARGET):
```
Comment (cdesiniotis) [inline validator/Makefile]: Same as above comment. Can this be removed?

File: Makefile (hunk 1 above)
Comment (tariq1890) [reply Makefile]: Sure, I am open to removing this. I only kept it as I haven't changed the make target names, only the names of the  tags that are generated as output of these make jobs

Comment (cdesiniotis) [reply Makefile]: We can remove this. The actual make target `push-ubi9` is defined in `multi-arch.mk` / `native.mk`.

File: validator/Makefile (hunk 2 above)
Comment (tariq1890) [reply validator/Makefile]: Done

File: Makefile (hunk 1 above)
Comment (tariq1890) [reply Makefile]: Done, please check now

**Review by cdesiniotis — APPROVED** [review]

**Pull request merged**